geoh5py/ui_json/input_file.py

```diff
@@ -489,9 +489,9 @@ def numify(cls, ui_json: dict[str, Any]) -> dict[str, Any]:
                     cls.ui_validation(value)
                 except tuple(BaseValidationError.__subclasses__()) as error:
                     raise JSONParameterValidationError(key, error.args[0]) from error
-
                 value = cls.numify(value)
 
+            # todo: problem here: cannot get a path to a normal path
```

Comment:
Good idea. Created an issue to implement GEOPY-1103

Reply:
The problem was not about that, 
It was about having a ".geoh5" in my text but not at the end.